src/index.js

```diff
@@ -55,6 +55,21 @@ const exported = {
         config.ACCESS_TOKEN = token;
     },
 
+    /**
+     * Gets and sets map's pixel ratio
+     * 
+     * @var {number} devicePixelRation
+     * @example
+     * mapboxgl.devicePixelRatio = 2
+     */
+    get devicePixelRatio() {
+        return config.DEVICE_PIXEL_RATIO;
+    },
+
+    set devicePixelRatio(devicePixelRatio: number) {
+        Object.defineProperty(config, 'DEVICE_PIXEL_RATIO', { value: devicePixelRatio });
```

Comment:
is there a reason to use `Object.defineProperty` here instead of simply assigning to `config.DEVICE_PIXEL_RATIO` the way that `accessToken` does? I'd say it's better to avoid having two separate ways of assigning to config properties here and unless there's a specific reason to favor `defineProperty`, I'd side with the simplest code possible.